### DISCUSSION TITLE
第5节课笔记(18班)

### DISCUSSION THREAD

**Discussion by mattheliu** [InternLM/Tutorial#149] [笔记]
请提交笔记链接
Please submit the link to the notes.

**Comment by yestolife** [comment]
[笔记：LMDeploy 的量化和部署](https://blog.csdn.net/Andytl/article/details/135578001)

**Comment by Heitao5200** [comment]
https://github.com/Heitao5200/my_blog/blob/master/docs/%E7%BB%84%E9%98%9F%E5%AD%A6%E4%B9%A0/%E5%A4%A7%E6%A8%A1%E5%9E%8B%E5%AE%9E%E6%88%98/TASK5.md

**Comment by 2001wjh** [comment]
https://www.yuque.com/wjh2001/rdvfen/dg19nrf69dd5h5t6?singleDoc# 《LMDeploy 大模型量化部署实践（笔记）》

**Comment by DrZhang1208** [comment]
[LMDeploy 大模型量化部署实践](https://blog.csdn.net/qq_42866033/article/details/135625863?spm=1001.2014.3001.5501)

**Comment by tackhwa** [comment]
[第5节课笔记+笔记](https://nutritious-chill-7a9.notion.site/LMDeploy-a9422127d8824298841a54140231a2cd?pvs=4)

**Comment by ggfengxuan** [comment]
[5.LMDeploy 的量化和部署笔记](https://v1y50l9n58a.feishu.cn/wiki/Dyxjwzqm1iF177kiDFfcOEAZnQc?from=from_copylink)

**Comment by xwhclaire** [comment]
[Lecture 5 LMDeploy大模型量化部署](https://github.com/xwhclaire/StudyPackages/blob/main/LLM%20training/Lecture%205%20LMDeploy%20%E5%A4%A7%E6%A8%A1%E5%9E%8B%E9%87%8F%E5%8C%96%E9%83%A8%E7%BD%B2%E5%AE%9E%E8%B7%B5.md)

**Comment by scorpionfree98** [comment]
[第五节课笔记+作业](http://t.csdnimg.cn/TDnvn)

**Comment by ZMcc18** [comment]
https://www.yuque.com/zixul/avxnzn/fwttpmeg4niheegy?singleDoc# 《lesson5 LMDeploy 的量化和部署 作业&笔记》

**Comment by mattheliu** [comment]
https://github.com/mattheliu/gitblog/issues/5
